### ISSUE TITLE
Feature request: Microphone support for Seaman 2

### ISSUE THREAD

**Issue by InVerse199** [jackun/USBqemu-wheel#66]
Would it be possible to add mic support to this underappreciated gem? It requires a mic to play. I can send the ROM (Japanese release) if that helps. Thanks a lot!

**Comment by jackun** [comment]
It's a WIP and little weird controller so configure mic in `Singstar` config dialog and then select `Seamic`. Maybe works.

**Comment by InVerse199** [comment]
Great, it now recognises the device. Now I just need to figure out how to navigate the menus from here. Thankyou very much for the reply!

**Comment by jackun** [comment]
Ah, also iirc "steering" for X-axis and "throttle" Y-axis of the stick.

**Comment by InVerse199** [comment]
Would I need an analog input to control those parameters?

**Comment by jackun** [comment]
Yeah, currently it's not setup for it and the "input system" has been pretty much hard-coded for wheels. Left/right _can_ be mapped to keyboard etc but Y-axis, up/down, is either always down or up.

**Comment by InVerse199** [comment]
That's fine, I'm sure I'll find a way to get it up & running.

**Comment by sonik-br** [comment]
Hi. I do have the ps2 controller for Seaman.
I can do some tests with it if needed.
BTW it works in windows. Just plug and play. Even the mic works. So it might be a generic hid controller?

![image](https://user-images.githubusercontent.com/16923826/144052540-31dc7161-6557-4af0-9f2d-f477269be83d.png)
